### PR TITLE
Small update for duckling and docker

### PR DESCRIPTION
**Pull request by minhtuev**
1. Relax the constraint to find duckling pid
2. Update docs with port mapping for duckling docker
3. Docker container does not need mindmeld config
4. ES cluster name to be mindmeld

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -67,6 +65,7 @@ RUN pip3 install --upgrade pip
 RUN pip3 install -U mindmeld
 RUN pip3 install click-log==0.1.8
 
+ARG REBUILD_BLUEPRINT=unknown
```
Comment (vijay120) [inline docker/Dockerfile]: what is this?

Comment (minhtuev) [reply docker/Dockerfile]: This is a trick to help rebuild docker container after a specific step

https://stackoverflow.com/questions/35154219/rebuild-docker-image-from-specific-step

Comment (minhtuev) [reply docker/Dockerfile]: Otherwise, Docker will use a cached version of the assistant

Comment (vijay120) [reply docker/Dockerfile]: :)

**Pull request merged**